Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: popup panel for connection status messages

### Description (*)

This PR adds a simple reusable popup panel that lives in the SettingsPanelCanvas inside the DontDestroyOnLoadScene and uses it to display messages to players related to connection status. Connection status messages are now published through a new message channel instead of being handled by LobbyUIMediator. Finally, a manager is added, which subscribes to this message channel and calls the PopupPanel when needed.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2677](https://jira.unity3d.com/browse/MTT-2677)
### Manual testing scenarios

1. Start a game without linking the project id to see a message about anonymous sign-in failing
2. Start a game with one or more clients and disconnect or pause the host to disconnect clients or make them timeout and see the corresponding messages
3. Try to join a full lobby to see another message
### Questions or comments

### Contribution checklist
